index.html

```diff
@@ -0,0 +1,286 @@
+                sign-ups required</p>
+              <div>
+                <div class="skills">
+                  <div class="skills__container">
+                    <p>HTML</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>CSS</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>JavaScript</p><br/>
+                  </div>
+              </div>
+              <div class="Portfolio__project__layout">
+                <div class="Portfolio__project__design">
+                  <div class="Portfolio__project__content">See Project </div>
+                </div>
+              </div>
+            </div>
+            </div>
+          </section>
+          <section>  
+          <div class="Portfolio__About__MySelf">
+            <h4 class="About__MySelf__header">About Myself</h4>
+            <div class="About__MySelf__Content">
+              <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="About__MySelf__layout">
+              <div class="About__MySelf__design">
+                <div class="About__MySelf__Connect">LET'S CONNECT</div>
+              </div>
+              <div class="social-media">
+              <ul>
+                <li><a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a></li>
+                <li><a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a></li>
+                <li><a href=""><img src="img/m_icon.png" alt="Medium icon" /></a></li>
+                <li><a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a></li>
+                <li><a href=""><img src="img/finger_c.png" alt="finger icon" /></a></li>
+            
+            </ul>
+           </div>
+           <div class="resume__tag">
+            <button class="resume__button"><p>Get my resume</p></button>
+           </div>
+        <div>
+          <ul>
+            <li class="Languages">
+              <h3>Languages</h3>
+               <img class="framework__arr" src="img/about_page/arr_d.svg"></div>
+                <ul>
+                   <li><div class="resume__coding">
+                     <img src="img/about_page/javascript.png" alt="">
+                     <p class="resume__coding_par">JavaScript</p></div>
+                   </li>
+                   <li><div class="resume__coding">
+                      <img src="img/about_page/html.png" alt="">
+                      <p class="resume__coding_par">HTML</p></div>
+                    </li>
+                    <li><div class="resume__coding">
+                       <img src="img/about_page/css.png" alt="">
+                       <p class="resume__coding_par">CSS</p></div>
+                    </li>
```

Comment:
- [ ] Kindly

Your design
![image](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/81932478/ce2617a6-80a9-49b5-af63-24be1762fadc)
![image](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/81932478/f204256f-1d4e-4f27-a081-66b9507fe040)

Given design
![image](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/81932478/1f5f9a32-4dd9-459b-9d0e-aea90fc155c2)
![image](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/81932478/d604d2c1-c895-44e9-a7f5-e4a5ca9ee8cf)

### Note that these required changes are because at `375` px your design is not matching the given design as per requirement. Kindly check the [requirements here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)